public/app/features/dashboard/components/DashChat/DashChatModal.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Modal } from '@grafana/ui';
+import { css } from '@emotion/css';
+import { ChatDashboardModalProps } from './types';
+import { Chat } from 'app/features/chat/Chat';
+
+export const DashboardChatModal: React.FC<ChatDashboardModalProps> = ({ dashboard, onDismiss, onSaveSuccess }) => {
```

Comment:
No need to include the `onSaveSuccess` if it is not being used

Reply:
Done